scripts/collect_workloads.py

```
@@ -788,12 +822,63 @@ def run_sglang_mode(
+                        try:
+                            child.unlink()
+                            _purged += 1
+                        except OSError:
+                            pass
+                print(f"  Purged {_purged} pre-driver (warmup) dump entries.")
+
+                print(
+                    f"  External driver: {external_driver_cmd!r} "
+                    f"(timeout={external_driver_timeout}s, port={_server_port})"
+                )
+                rc = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)